substrate/client/authority-discovery/src/interval.rs

```diff
@@ -37,14 +38,20 @@ impl ExpIncInterval {
 	/// Create a new [`ExpIncInterval`].
 	pub fn new(start: Duration, max: Duration) -> Self {
 		let delay = Delay::new(start);
-		Self { max, next: start * 2, delay }
+		Self { start, max, next: start * 2, delay }
 	}
 
 	/// Fast forward the exponentially increasing interval to the configured maximum.
 	pub fn set_to_max(&mut self) {
 		self.next = self.max;
 		self.delay = Delay::new(self.next);
 	}
+
+	/// Fast forward the exponentially increasing interval to the configured start.
```

Comment:
nit: Fast forward -> rewind?
```suggestion
	/// Rewind the exponentially increasing interval to the configured start.
```